specs/eip4844/polynomial-commitments.md

```diff
@@ -226,12 +277,64 @@ def compute_kzg_proof(polynomial: Sequence[BLSFieldElement], z: BLSFieldElement)
     return KZGProof(g1_lincomb(bit_reversal_permutation(KZG_SETUP_LAGRANGE), quotient_polynomial))
 ```
 
+#### `compute_aggregated_poly_and_commitment`
+
+```python
+def compute_aggregated_poly_and_commitment(
+        blobs: Sequence[Blob],
+        kzg_commitments: Sequence[KZGCommitment]) -> Tuple[Polynomial, KZGCommitment]:
+    """
+    Return the aggregated polynomial and aggregated KZG commitment.
+    """
+    # Generate random linear combination challenges
+    r = hash_to_bls_field(blobs, kzg_commitments)
```

Comment:
I've been informed that blobs and polynomials are the same.

Reply:
Not anymore :)

Reply:
Ideally the polynomial-commitments.md file has no reference to what a blob is (just polynomials), I was thinking to push that to a separate PR, but we might aswell do it in the same PR now :)

Reply:
> Ideally the polynomial-commitments.md file has no reference to what a blob is (just polynomials), I was thinking to push that to a separate PR, but we might aswell do it in the same PR now :)

while this seems like a sensible idea in principle, I guess we are not doing it in this PR, correct? it would be a pretty substantial refactor.